IntegrationTests.MediumTrust/Sandboxing/PermissionSets.cs

```diff
@@ -48,7 +47,7 @@ public static IPermission[] GetMediumTrust (string appDir, string originHost)
                    new SecurityPermission (SecurityPermissionFlag.Assertion | SecurityPermissionFlag.Execution | SecurityPermissionFlag.ControlThread
                                            | SecurityPermissionFlag.ControlPrincipal | SecurityPermissionFlag.RemotingConfiguration),
                    new SmtpPermission (SmtpAccess.Connect),
-                   new SqlClientPermission (PermissionState.Unrestricted),
+                   //new SqlClientPermission (PermissionState.Unrestricted),
```

Comment:
```suggestion
                   new SqlClientPermission (PermissionState.Unrestricted),
```

Reply:
This project is only built in .net framework which still contains the relevant type. What was missing was the using directive.